fonts/circular_3letters_monogram/font.json

```diff
@@ -0,0 +1,117 @@
+{
+    "name": "Circular 3 Letters Monogram",
+    "description": "Enter 3 letters to get a circular three letters monogram. You can add a frame, but adding one of the following :(,),{,},[,],,,.,;,:,<,>\nFor instance enter {ABC.",
```

Comment:
I love the amount of frames that come with the font. In the description I would remove the commas as they make it a bit harder to read.

Typo: `You can add a frame, but adding one of the following :` -> `You can add a frame by adding one of the following:`

Reply:
<img width="1677" height="1015" alt="image" src="https://github.com/user-attachments/assets/af68a758-92a1-4fc2-b97d-ba94c4a7d431" />
 I could make Invercelia a little bit nicer with rebellious people like you. But then two is two, not three.
 I simply added a proper unknown character. If you type more than 2 letters, first and last are rendered, the other ones are ignored (no medi form in the font only init and fina)
 For Circular 3 letters, it is different, as there is a medi form, if you type 4 letters, 2 and 3 are superposed in the middle. I'm afraid rebellious people will have to learn to count up to 3! I added a warning in the json file.

Reply:
Great, thank you! Now it's working fine :)